client/components/foldable-card/index.jsx

```diff
@@ -41,7 +41,7 @@ var FoldableCard = React.createClass( {
 			onClose: noop,
 			cardKey: '',
 			icon: 'chevron-down',
-			isExpanded: false
+			isExpanded: false,
```

Comment:
I couldn't find this explicit in our style guide. There was one example that included a list and it had the trailing comma. Most of the time I see them I feel like the don't have the trailing comma, but Babel handles it gracefully, so maybe this boils down to personal preference here.